### PR TITLE
Product-page

### PR DESCRIPTION
**Pull request by hieunguyen64**
Added a single product page. At homepage, when clicking on any product image, it will redirect to the single product page.
Removed class search-container2.
The logo will now be responsive.
Logo, search bar and cart button will now be on a row.
Optimized code.

### CODE REVIEW

File: Product-page/Product-page.html [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+                            <img src="../images/Tango Company/Office chair/Essentials-Black.jpg" alt="Product 5" />
+                            <span class="border site-btn btn-span">OFFICE STAR</span>
+                        </a>
+                    </div>
+                    <div class="col-2 product pt-4">
+                        <a href="Product-page.html">
+                            <img src="../images/Tango Company/Office chair/Executive-Office.jpg" alt="Product 6" />
+                            <span class="border site-btn btn-span">PROFORM</span>
+                        </a>
+                        </a>
```
Comment (thienhophu) [inline Product-page/Product-page.html]: remove this

File: Product-page/css/Product-page.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#addtocart-btn{
+    font-size: 20px;
+    width: 120px;
+}
+#favorites-btn{
```
Comment (thienhophu) [inline Product-page/css/Product-page.css:8]: extra line

File: index.html [hunk 3]
```diff
@@ -146,44 +141,65 @@ <h4>Office chairs</h4>
...
+                        </a>
                     </div>
                     <div class="col-2 product pt-4">
-                        <img src="images/Tango Company/Office chair/Executive-Office.jpg" alt="Product 6" />
-                        <span class="border site-btn btn-span">PROFORM</span>
+                        <a href="Product-page/Product-page.html">
+                            <img src="images/Tango Company/Office chair/Executive-Office.jpg" alt="Product 6" />
+                            <span class="border site-btn btn-span">PROFORM</span>
+                        </a>
+                        </a>
```
Comment (thienhophu) [inline index.html]: remove this

**Review by thienhophu — CHANGES_REQUESTED** [review]

File: Product-page/Product-page.html (hunk 1 above)
Comment (hieunguyen64) [reply Product-page/Product-page.html]: done

File: Product-page/css/Product-page.css (hunk 2 above)
Comment (hieunguyen64) [reply Product-page/css/Product-page.css:8]: done

File: index.html (hunk 3 above)
Comment (hieunguyen64) [reply index.html]: done

**Review by thienhophu — APPROVED** [review]

**Pull request merged**